src/data/faq_de.json

```
@@ -102,6 +102,148 @@
+                        "anchor": "vac_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Zu Beginn des Jahres wurde durch den Europäischen Rat beschlossen, einen interoperablen und standardisierten Impfnachweis für medizinische Zwecke auf den Weg zu bringen. Deutschland setzt diese europäische Entscheidung nun schnell um, um allen Bürgern, die dies wünschen, einen digitalen Impfnachweis zur Verfügung stellen zu können. Mittlerweile hat die Europäische Kommission auch einen Verordnungsentwurf zur Regelung eines EU-weiten Anerkennungsrahmens für (digitale) Zertifikate für Impfungen, Tests und für Personen mit COVID-19-Heilstatus vorgelegt, um die Reisefreizügigkeit zu erleichtern. Das grüne Zertifikat der EU bildet den Rechtsrahmen für die Lösungen der Mitgliedsstaaten."
+                        ]
+                    },
+                    {
+                        "title": "Wer kann einen elektronischen Impfnachweis erhalten? Wie erhalte ich ein Impfzertifikat? Wie kann ich, wenn ich bereits vollständigen Impfschutz habe, den Status nachträglich in die Corona-Warn-App übertragen?",
+                        "anchor": "vac_cert_who",
+                        "active": true,
+                        "textblock": [
+                            "Alle Personen, die einen Anspruch auf eine Impfung haben, können sich ergänzend zum analogen Impfnachweis auch einen digitalen Impfnachweis ausstellen lassen.",
+                            "Sie können gegebenenfalls ein Corona-Impfzertifikat von der Arztpraxis oder dem Impfzentrum erhalten, wo die Impfung vorgenommen wurde. Fragen Sie vor Ort ganz einfach nach. Die Impfzentren und die Arztpraxen sind jedoch nicht dazu verpflichtet, Impfzertifikate auszustellen.",
+                            "Es werden verschiedene Möglichkeiten geprüft, um auch nachträglich digitale Impfnachweise für Corona-Impfungen zu erstellen. Grundsätzlich soll die nachträgliche Ausstellung dort erfolgen, wo man geimpft worden ist. Wenn in den Impfzentren entsprechende Kontaktdaten vorliegen, sollen die QR-Codes möglichst automatisch per Post zugesandt werden. Außerdem könnten Ärztinnen und Ärzte sowie Apothekerinnen und Apotheker nachträglich Impfnachweise ausstellen."
+                        ]
+                    },
+                    {
+                        "title": "Kann ein vollständiger Impfstatus in der App als Berechtigung zum Reisen dienen?",
+                        "anchor": "vac_cert_privileges_travel",
+                        "active": true,
+                        "textblock": [
+                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab.",
+                            "Sie können unter anderem dieses <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Instrument</a> benutzen, um ihre Reise innerhalb der EU zu planen."

```

I think it's okay to use "Tool" here in German. "Instrument" sounds a bit weird IHMO.
```suggestion
                            "Sie können unter anderem dieses <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Tool</a> benutzen, um ihre Reise innerhalb der EU zu planen."
```

---

Agreed that "Instrument" is not a good term. How about simply "Webseite"?

---

@fynngodau Agreed & changed. 👍

---

Now you changed your suggestion to "Website", and that's not a German word either ;)

---

Sorry, was quite an intense day...